src/components/table/expandable/row/style.css

```
@@ -2,11 +2,43 @@
 	display: table-row;
 	border: 1px solid rgb(var(--smoothly-default-color));
 	cursor: pointer;
+	box-sizing: border-box;
 }
-:host:nth-child(even) {
-	background-color: rgb(var(--smoothly-default-tint));
+.hide {
+	display: none;
 }
-:host:hover {
+.slot-detail {
+	position: relative;
 	background-color: rgb(var(--smoothly-default-color));
+	width: 104%;
+	left: calc(-52% + 50%);
```

You're using the same unit, hence you don't `calc`.
```suggestion
	left: -2%;
```